### PR TITLE
influxdb sender中增加自动创建database和retention policy的逻辑

### PR DESCRIPTION
**Pull request by DangHexuan**
## Changes
   
  - [x] 增加autoCreate选项控制是否自动创建database
  - [x] 增加influxdb_retention_duration选项，在autoCreate为true的时候先创建database，然后再创建retention policy 
 
## Reviewers

  - [ ] @wonderflow  please review
  - [ ] @andrewei1316  please review

## Wiki Changes
 
  - options1...
  - options2...
    
## Checklist
   
   - [ ] Rebased/mergable
   - [ ] Tests pass
   - [ ] Wiki updated

### CODE REVIEW

File: sender/influxdb_sender.go [hunk 1]
```diff
@@ -52,6 +57,7 @@ func NewInfluxdbSender(c conf.MapConf) (s Sender, err error) {
 	if err != nil {
 		return
 	}
+	autoCreate, _ := c.GetBoolOr(KeyInfluxdbAutoCreate, false)
```
Comment (wonderflow) [inline sender/influxdb_sender.go]: 感觉默认为true舒服

Comment (DangHexuan) [reply sender/influxdb_sender.go]: ok

**Comment by DangHexuan** [comment]
@wonderflow  `跟前端交互的配置项` 怎么加？

**Comment by wonderflow** [comment]
LGTM

**Comment by andrewei1316** [comment]
LGTM

**Pull request merged**